bin/dsn.cmake

```diff
@@ -231,7 +231,7 @@ endmacro(ms_replace_compiler_flags)
 function(ms_check_cxx11_support)
     if(UNIX)
         include(CheckCXXCompilerFlag)        
-        CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
+        CHECK_CXX_COMPILER_FLAG("-std=c++1y" COMPILER_SUPPORTS_CXX11)
```

Comment:
Actually we are using C++14 features, in which we can use move semantic in lambda capture list, such as things like this: "[ b = std::move(c)]".
Well, the compile flag "-std=c++11" works fine on my desktop with the g++ version of 4.8. But on my laptop with the g++ version of 5.1, "-std=c++11" won't compile.